src/test/scala/wdl4s/wdl/values/WdlValueSpec.scala

```diff
@@ -50,7 +50,7 @@ class WdlValueSpec extends FlatSpec with Matchers {
     (WdlFloat(Double.MinValue), "-1.7976931348623157E308", { d: Double => d.isNegInfinity }))
 
   forAll(wdlFloatSpecials) { (wdlValue, rawString, validateFloat) =>
-    it should s"convert a special ${wdlValue.typeName} to/from raw string '$rawString'" in {
+    it should s"convert a special ${wdlValue.typeName} to/from raw string '$rawString'" ignore {
```

Comment:
I think I'm just going to delete these tests unless there are any objections.  The tested behavior has been eliminated and I think the world is a better place for that.